apps/cloud/setup.py

```diff
@@ -22,17 +22,20 @@
         'odc_aio',
         'odc_ppt',
         'odc_thredds',
+        'odc_azure',
         "click",
     ],
 
     extras_require={
         'GCP': ['google-cloud-storage'],
-        'THREDDS': ['thredds_crawler', 'requests']
+        'THREDDS': ['thredds_crawler', 'requests'],
+        'AZURE': ['azure-storage-blob']
```

Comment:
doesn't really matter, but for the future: Python is ok with trailing commas and it helps with future diffs.